.github/ISSUE_TEMPLATE/installation_issue.md

```diff
@@ -11,6 +11,9 @@ labels: "installation"
 
 - [ ] I have looked through [issues labeled "installation"](https://github.com/geopandas/geopandas/labels/installation) in the geopandas repo.
 
+- [ ] If using conda, I have looked through [issues on conda-forge geopandas-feedstock](https://github.com/conda-forge/geopandas-feedstock) (or
```

Comment:
Suggestion: "If using conda and the third party channel conda-forge,..."

The reason for that is b/c there is a `geopandas` package on defaults now.